[2023-04-09T00:32:50.848+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.process_user manual__2023-04-09T00:32:45.588973+00:00 [queued]>
[2023-04-09T00:32:50.854+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.process_user manual__2023-04-09T00:32:45.588973+00:00 [queued]>
[2023-04-09T00:32:50.855+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:32:50.856+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-09T00:32:50.857+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:32:50.865+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): process_user> on 2023-04-09 00:32:45.588973+00:00
[2023-04-09T00:32:50.871+0000] {standard_task_runner.py:55} INFO - Started process 9666 to run task
[2023-04-09T00:32:50.875+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'process_user', 'manual__2023-04-09T00:32:45.588973+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmp6of1w641']
[2023-04-09T00:32:50.877+0000] {standard_task_runner.py:83} INFO - Job 60: Subtask process_user
[2023-04-09T00:32:50.934+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.process_user manual__2023-04-09T00:32:45.588973+00:00 [running]> on host c12a518d534c
[2023-04-09T00:32:50.990+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T00:32:45.588973+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T00:32:45.588973+00:00
[2023-04-09T00:32:51.008+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-09T00:32:51.018+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=process_user, execution_date=20230409T003245, start_date=20230409T003250, end_date=20230409T003251
[2023-04-09T00:32:51.046+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T00:32:51.065+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
